keras_nlp/tokenizers/sentence_piece_tokenizer.py

```diff
@@ -169,6 +169,11 @@ def get_vocabulary(self) -> List[str]:
 
     def id_to_token(self, id: int) -> str:
```

Comment:
Should we add the same to word piece tokenizer and byte pair tokenizer?

From [your comment](https://github.com/keras-team/keras-nlp/issues/1036#issuecomment-1547729640) on this issue it sounds like they already have more reasonable behavior, but still a uniform error message seems like it would be nice.

Reply:
sure. I will add it.

Reply:
in `WordPieceTokenizer` it worked fine.
but in `BytePairTokenizer` it's a bit tricky because the vocabularies are dictionaries mapping token to id. and may be the user may use a dictionary of ids starting from 1 ( The example in docstring uses ids starting from 1) or may be ids of non Consecutive numbers -I am not sure if a user will do that, but he can-
so it will be better for `BytePairTokenizer` to do that :
```python
def id_to_token(self, id: int) -> str:
    keys = self.get_vocabulary()
    for token in keys:
        if self.vocabulary[token] == id:
            return token
    raise ValueError("Recieved `id` is out of the vocabulary")
```
or should I just assume that the vocabulary always start at 0

Reply:
I added it like this for now. and I am waiting for review.